crates/bevy_pbr/src/light_probe/environment_map.rs

```diff
@@ -114,6 +116,80 @@ pub struct EnvironmentMapLight {
     pub affects_lightmapped_mesh_diffuse: bool,
 }
 
+impl EnvironmentMapLight {
```

Comment:
Needs docs for the new functions. Esp since EnviornmentMapLight is supposed to have diffuse/specular components that get run through a bunch of sampling and multiscattering stuff, having a single "color" is a bit confusing. Please add some docs and I guess note that it's a bit of a hack? Something along these lines.